addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -424,7 +424,8 @@ var NumericField = InputField.extend({
      * @private
      */
     _formatValue: function (value) {
-        if (this.mode === 'edit' && this.nodeOptions.type === 'number') {
+        var noFormat = !!JSON.parse(this.nodeOptions.no_format || false);
```

Comment:
as @Arcasias would say, this is 2020, you can use const

Reply:
It was still 2019 when I wrote this :D